.github/workflows/ci.yml

```diff
@@ -195,7 +188,7 @@ jobs:
           echo "$modifications"
 
           if [[ "$modified" = "true" ]]; then
-            echo "::error ::Rebuilt WASM doesn't match committed WASM. Please rebuild and commit."
+            echo "::error ::Rebuilt Wasm doesn't match committed Wasm. Please rebuild and commit."
```

Comment:
I think maybe this verification is failing because I built it on Windows?

Reply:
I rebuilt on Linux and maybe we'll see if that works. We'll need a better solution than this in the future.